src/c4_framework/mod.rs

```diff
@@ -1,4 +1,4 @@
-/// Braindump (revise later):
+/// Brain dump (TODO revise later):
```

Comment:
Okay, but just realize this is under active development. Spending time on things like this may feel wasteful later.